utils/docker/Dockerfile.maldet.centos.7

```diff
@@ -0,0 +1,162 @@
+# Copyright 2018-2022 Intel Corporation
```

Comment:
This file is new:
```suggestion
# Copyright 2022 Intel Corporation
```

Reply:
I've always assumed that's how it worked.